src/main/java/Parser.java

```diff
@@ -0,0 +1,3 @@
+public class Parser{
```

Comment:
This is nitpicking but don't forget the space to follow coding standards. In the future can check through.